wolox-training/ap-swift
# Suggest book

## Summary ##
**Trello card**: https://trello.com/c/LfBx8EED/22-vista-para-sugerir-un-nuevo-libro
**Trello card**: https://trello.com/c/tXZv6KsJ/23-input-fields

Suggest a new book view

## Test Plan ##

  * Open the app
  * Login with Google
  * The main tab bar view should appear after a successfull login.
  * Press over Suggest Tab item
  * The Suggest Book View view should with book details to complete

## Screenshots ##

### iPhone 7s

![login-view](https://cloud.githubusercontent.com/assets/25587882/23802748/0331ed02-0593-11e7-9251-6f6695c20681.png)
![libraryview](https://cloud.githubusercontent.com/assets/25587882/23802756/091eb056-0593-11e7-843f-de105aec860d.png)
![screen shot 2017-03-14 at 11 08 31 am](https://cloud.githubusercontent.com/assets/25587882/23904478/c12004a0-08a6-11e7-9a91-f93c1233d391.png)
![screen shot 2017-03-14 at 11 09 05 am](https://cloud.githubusercontent.com/assets/25587882/23904480/c40a88d4-08a6-11e7-854f-eaa86d3a429c.png)

## Screencast ##

![suggestbook2](https://cloud.githubusercontent.com/assets/25587882/23904489/cd38c024-08a6-11e7-98dd-4e89e67e8b0f.gif)

## Known issues ##

The submit button has not the gradient implemented
